src/libproc_macro/lib.rs

```diff
@@ -255,6 +255,13 @@ macro_rules! diagnostic_method {
 }
 
 impl Span {
+    /// A span that resolves at a unique site. Each call to this method will return a new unique
+    /// span that is inaccessible from other sites.
```

Comment:
The documentation should mention "gensym" here in some fashion since that's familiar for some folks who would use this.

Reply:
Good idea.